code/datums/station_traits/positive_traits.dm

```diff
@@ -270,3 +270,59 @@
 	// Put our filthy fingerprints all over the contents
 	for(var/obj/item/item in wallet)
 		item.add_fingerprint(living_mob, ignoregloves = TRUE)
+
+/datum/station_trait/cybernetic_revolution
+	name = "Cybernetic Revolution"
+	trait_type = STATION_TRAIT_POSITIVE
+	show_in_report = TRUE
+	weight = 1
+	report_message = "The new trends in cybernetics have come to the station! Everyone has some form of cybernetic implant."
+	/// List of all job types with the cybernetics they should receive.
+	var/static/list/job2cybernetic = list(
+		/datum/job/assistant = /obj/item/organ/internal/heart/cybernetic,
+		/datum/job/atmospheric_technician = /obj/item/organ/internal/cyberimp/mouth/breathing_tube,
+		/datum/job/bartender = /obj/item/organ/internal/liver/cybernetic/tier3,
+		/datum/job/botanist = /obj/item/organ/internal/cyberimp/chest/nutriment,
+		/datum/job/captain = /obj/item/organ/internal/heart/cybernetic/tier3,
+		/datum/job/cargo_technician = /obj/item/organ/internal/stomach/cybernetic/tier2,
+		/datum/job/chaplain = /obj/item/organ/internal/cyberimp/brain/anti_drop,
+		/datum/job/chemist = /obj/item/organ/internal/liver/cybernetic/tier2,
+		/datum/job/chief_engineer = /obj/item/organ/internal/cyberimp/chest/thrusters,
+		/datum/job/chief_medical_officer = /obj/item/organ/internal/cyberimp/chest/reviver,
+		/datum/job/clown = /obj/item/organ/internal/cyberimp/brain/anti_stun,
+		/datum/job/cook = /obj/item/organ/internal/cyberimp/chest/nutriment/plus,
+		/datum/job/curator = /obj/item/organ/internal/eyes/robotic/glow,
+		/datum/job/detective = /obj/item/organ/internal/lungs/cybernetic/tier3,
+		/datum/job/doctor = /obj/item/organ/internal/cyberimp/arm/surgery,
+		/datum/job/geneticist = /obj/item/organ/internal/fly,
+		/datum/job/head_of_personnel = /obj/item/organ/internal/eyes/robotic,
+		/datum/job/head_of_security = /obj/item/organ/internal/eyes/robotic/thermals,
+		/datum/job/janitor = /obj/item/organ/internal/eyes/robotic/xray,
```

Comment:
“Admiral, who do we give the strongest and most advanced cybernetics?”
“Give it to the janitor, HoS can have thermals or smthing idk”

Reply:
you need to be able to see the whole station's cleanliness